HISTORY.md

```diff
@@ -72,6 +72,8 @@
 * fix(windows): Track modifier changes in UWP apps (#4468)
 * fix(common/resources): Fix help.keyman.com path for commit (#4469)
 * fix(common): create GitHub comments serially (#4472)
+* feat(common/models): mid-context suggestions & reversions, fix(common/models): correction-search SMP issues (#4427)
```

Comment:
This PR hasn't been merged yet and should not be mentioned.

Reply:
Yeah, saw that as I looked at HISTORY.md and then forgot to fix it while fixing up all the versioning merge conflicts yay package.json :(